client/coral-framework/modules/i18n/i18n.js

```diff
@@ -58,8 +60,9 @@ class i18n {
         return key;
       }
 
-      const val = String(translation);
+      let val = String(translation);
       if (val) {
+        val = val.replace('/\{0\/g', att);
```

Comment:
I tested this line and it didn't seem to work as desired. Should it be:

`val = val.replace(/\{0\}/g, att);`?